packages/components/src/styles/components/modal.scss

```diff
@@ -8,13 +8,18 @@
 //
 
 .hds-modal {
+  all: revert; // this must be first
   z-index: 50;
-  flex-direction: column;
+  box-sizing: inherit;
```

Comment:
I don't think you need this 🤔

Reply:
Yeah looks like you are right. I was having issues with boxsizing still so added that but it could have been that the style change hadn't compiled correctly or something when I last checked. (I keep having to rebuild and restart Showcase and Web to see changes and/or fix errors that occur.)